ai_genomics/analysis/tag_evaluation/make_tag_validation_dataset.py

```diff
@@ -0,0 +1,130 @@
+"""Script to 1) extract openalex concepts from openalex sample and
+2) generate validation dataset to compare tags across
+sample of sample of texts with DBpedia and OpenAlex tags."""
+
+from ai_genomics.getters.data_getters import load_s3_data, get_s3_dir_files, save_to_s3
+
+from ai_genomics import bucket_name, logger
+from typing import List, Union
+import pandas as pd
+import requests
+
+SAMPLES_DIR = get_s3_dir_files(bucket_name, "inputs/ai_genomics_samples/")
+OA_SAMPLE = load_s3_data(
+    bucket_name, "inputs/ai_genomics_samples/ai_genomics_openalex_samples.csv"
+)
+
+
+def make_openalex_api_url(work_url: str) -> str:
+    """Convert openalex sample work id into json-compatible url."""
+    return f"https://api.openalex.org/works/{work_url.split('/')[-1]}"
+
+
+def get_openalex_sample_concepts(oa_sample: pd.DataFrame = OA_SAMPLE) -> pd.DataFrame:
+    """Calls OpenAlex API to get concepts per OpenAlex sample work id.
+        Returns OpenAlex sample with associated concepts, concept
+        scores and concept levels.
+
+    Args:
+        oa_sample: DataFrame of OpenAlex text samples
+    """
+    oa_sample["api_url"] = oa_sample["work_id"].apply(make_openalex_api_url)
+
+    oa_tags = []
+    for sample in list(oa_sample["api_url"]):
```

Comment:
this should work the same
```suggestion
    for sample in oa_sample["api_url"]: 
```